scripts/init.sh

```diff
@@ -67,7 +71,10 @@ fi
 
 run "$EXEC_CMD"
 
-run "cp ./setup/sirius/database/keystore/* ${DATA_DIR}/database/keystore/"
+if [ -z "${STANDALONE_NET}" ]; then
```

Comment:
hm why checking for STANDALONE_NET and then handling Sirius private keys?

Reply:
what the "-z" flag means?

Reply:
`-z` means `NOT standalone` therefore keys are ONLY copied in case of `sirius`

Reply:
hm ok, couldn't u do: "if [ ${SIRIUS_NET}] "? because this impl will break the moment we add another network

Reply:
IRL UPDATE: @ggarri will make sure this doesn't break when new network is added